### PR TITLE
chore(package): add build:w3c-debug script

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,10 +73,12 @@
   "scripts": {
     "babel:build": "babel src -d js --source-maps -q",
     "babel:watch": "babel src -d js --watch --source-maps",
-    "build:w3c": "node ./tools/builder.js --profile=w3c",
-    "build:geonovum": "node ./tools/builder.js --profile=geonovum",
     "build:components": "npm run copydeps && npm run babel:build",
+    "build:geonovum-debug": "node ./tools/builder.js --profile=geonovum --debug",
```
Comment (saschanaz) [inline package.json]: How about this to reduce code duplication?

```suggestion
    "build:geonovum-debug": "npm run build:geonovum -- --debug",
```

Comment (marcoscaceres) [reply package.json]: TIL :)

**Comment by marcoscaceres** [comment]
Did the same for the w3c ones.

**Comment by sidvishnoi** [comment]
Why did we add these scripts? Why not just pass the `--debug` when running these scripts?

**Comment by marcoscaceres** [comment]
--debug It's not obvious, it's a hidden "gem" 💎

**Comment by saschanaz** [comment]
To save 4 characters for each time? 😄

**Pull request merged**